app/models/grda_warehouse/project_group.rb

```diff
@@ -38,6 +38,12 @@ class ProjectGroup < GrdaWarehouseBase
       viewable_by(user)
     end
 
+    scope :text_search, -> (text) do
+      return none unless text.present?
+  
+      where(arel_table[:name].lower.matches("%#{text.downcase}%"))
```

Comment:
I managed to implement most of the features on the tracker list, but I can't figure out a clean way to write a query to search on Project Names. It would be great if I could get some help with this.